compiled_starters/zig/spawn_redis_server.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+#
+# DON'T EDIT THIS!
+#
+# CodeCrafters uses this file to test your code. Don't make any changes here!
+#
+# DON'T EDIT THIS!
+set -e
+
+exec zig build run
```

Comment:
@ippsav let's forward arguments to the program here using "$@" (it's required for extensions like Replication / RDB persistence)

Reply:
it would be as just a matter of adding
```
exec zig build run -- $@
```

Reply:
I ll be adding it and tinker with the dependencies bit tomorrow and keep you updated !